model/delivery/task/LtiDeliveryCreationTask.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: siwane
+ * Date: 15/05/19
+ * Time: 21:43
+ */
+
+namespace oat\taoLtiConsumer\model\delivery\task;
+
+use oat\generis\model\OntologyAwareTrait;
+use oat\oatbox\extension\AbstractAction;
+use oat\tao\model\taskQueue\Task\TaskAwareInterface;
+use oat\tao\model\taskQueue\Task\TaskAwareTrait;
+use oat\taoDeliveryRdf\model\DeliveryAssemblyService;
+use oat\taoLtiConsumer\model\delivery\factory\LtiDeliveryFactory;
+
+class LtiDeliveryCreationTask extends AbstractAction implements \JsonSerializable
+{
+//    use TaskAwareTrait;
+    use OntologyAwareTrait;
+
+    /**
+     * @param $params
+     * @throws \common_exception_MissingParameter
+     * @return Report
+     */
+    public function __invoke($params)
```

Comment:
`array` type hint, or can it be null?
Below we assume that it's an array

Reply:
CAnnot must be compatible with interface